src/sbt-test/sbt-databricks/rest-calls/project/TestBuild.scala

```diff
@@ -1,3 +1,5 @@
+package sbtdatabricks
```

Comment:
can you remove this? Tests should mimic an environment outside of `sbtdatabricks`. if you remove the private from the `Context`... stuff above, it should be okay